src/App.vue

```diff
@@ -145,7 +145,7 @@ const canShowCollapseButton = computed(
           <img v-else :src="getImage('collapse-arrow.svg')" />
         </button>
       </div>
-      <WalletHeader v-if="sdkVersion === 'v3' && route.name !== 'requests'" />
```

Comment:
@shrinathprabhu why this is removed or replaced?

Reply:
Removed because older version auth sdk are not able to see Wallet header which has 2 important things
- Switch chain/network button
- OR Code button

So removed the `sdkVersion` check so header is available in every version